examples/shared/shared.rb

```diff
@@ -22,8 +22,11 @@ module Shared
   attr_accessor :write_policy, :policy, :client, :logger
 
   def init
+
     @@options = {
-      :port => 3000,
+      # setting host as localhost returns an error on Macs, so check env variable first
+      :host => ENV['AEROSPIKE_HOST'] || 'localhost',
+      :port => ENV['AEROSPIKE_PORT']&.to_i || 3000,
```

Comment:
The client already uses the `AEROSPIKE_HOSTS` environment variable. Do we really need to introduce two new, separate vars for host and port?